cat.md

```diff
@@ -120,22 +122,22 @@ Cat Semantics
           => #fun(LOT
           => #fun(ART
           => #fun(TAB
-          => call Vat . grab ILK URN THIS VOWADDR (-1 *Rat LOT) (-1 *Rat ART)
+          => call Vat . grab ILK URN THIS VOWADDR (0Wad -Wad LOT) (0Wad -Wad ART)
           ~> call Vow . fess TAB
-          ~> call Flip ILK . kick URN VOWADDR (TAB *Rat CHOP) LOT 0
+          ~> call Flip ILK . kick URN VOWADDR (TAB *Rad Ray2Rad(CHOP)) LOT 0Rad
```

Comment:
I think `(TAB *Rad Ray2Rad(CHOP))` isn't quite what we want...this conceptually results in some sort of "RadRad" (90 digit fixed point).

I think what we really want is `rmul(TAB, CHOP)`. I guess we'll need to add an `rmul` for `Rad` left args:
```
syntax Rad ::= rmul ( Rad , Ray ) [function]
// etc
```
Hopefully that doesn't clash with the existing `rmul` defined for `Wads`. If there's an easy way to do parametric typing in `K`, the really "correct" way to define `rmul` would be something like (making up a syntax here):
```
template <numeric_sort T>
syntax T ::= rmul ( T , Ray ) [function]
// def
```
so `rmul` always returns the type of its first argument.

Reply:
I've switched this to `rmulRad`.